### ISSUE TITLE
Redis 4.0 support

### ISSUE THREAD

**Issue by Sembiance** [sripathikrishnan/redis-rdb-tools#80]
So attempting to process a redis 4.0rc2 dump.rd file yields:

```
Traceback (most recent call last):
  File "/usr/bin/rdb", line 11, in <module>
    load_entry_point('rdbtools==0.1.7', 'console_scripts', 'rdb')()
  File "/usr/lib64/python2.7/site-packages/rdbtools-0.1.7-py2.7.egg/rdbtools/cli/rdb.py", line 79, in main
    parser.parse(dump_file)
  File "/usr/lib64/python2.7/site-packages/rdbtools-0.1.7-py2.7.egg/rdbtools/parser.py", line 304, in parse
    self.parse_fd(open(filename, "rb"))
  File "/usr/lib64/python2.7/site-packages/rdbtools-0.1.7-py2.7.egg/rdbtools/parser.py", line 309, in parse_fd
    self.verify_version(f.read(4))
  File "/usr/lib64/python2.7/site-packages/rdbtools-0.1.7-py2.7.egg/rdbtools/parser.py", line 697, in verify_version
    raise Exception('verify_version', 'Invalid RDB version number %d' % version)
Exception: ('verify_version', 'Invalid RDB version number 8')
```

Not sure if it's a major deal to add support for 4.0 dumps or not.

**Comment by Sembiance** [comment]
Updating parser.py from:
```if version < 1 or version > 7:```
to
```if version < 1 or version > 8:```

Seems to have fixed it. Not sure if I'm getting inaccurate data or not, but the format seems compatible.

**Comment by oranagra** [comment]
the format isn't compatible, that's why the rdb version was incremented.
changes are (if i didn't miss anything):
- 64 lengths bit support 
- sorted sets use raw double format
- redis modules support
- few additional aux fields

so, i suppose that if you don't have large objects, don't use modules, and don't use sorted sets... then skipping the version check can work for you.

i suppose proper rdb 8 support will be added in the near future.
*excluding modules, and maybe some updates to the memory usage calculator (which requires quite a lot of changes).

**Comment by oranagra** [comment]
fixed by #83
